test/integration/per-node-stream-metrics.test.js

```diff
@@ -45,19 +91,33 @@ describe('metricsStream', () => {
                 perNodeMetrics: {
                     enabled: true,
                     wsUrl: 'ws://127.0.0.1:' + wsPort1 + '/api/v1/ws',
-                    httpUrl: 'http://localhost/api/v1'
+                    httpUrl: 'http://localhost/api/v1',
+                    intervals: {
```

Comment:
I'm a bit confused about why these need to be set to 1000? Couldn't you use the default values and get rid of the parameter ` reportMiliseconds` in the StreamrMetrics constructor, thereby making the class more simple?